### PR TITLE
QUEST_LV_0200.tsv [Lines 2400-2500]

### PR DESCRIPTION
**Pull request by Flevalt**

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -2402,10 +2402,10 @@ QUEST_LV_0200_20150317_002401	You've lighted up the brazier as Bokor Editor requ
 QUEST_LV_0200_20150317_002402	Talk with the Alchemist Saliamonas
 QUEST_LV_0200_20150317_002403	The Alchemist Saliamonas is waiting for someone's help at the petrified city.
```
Comment (f-tsang) [inline QUEST_LV_0200.tsv]: Petrified City is a place name. It should be capitalized here.

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -2413,18 +2413,18 @@ QUEST_LV_0200_20150317_002412	Saliamonas asked you to defeat Elite Monsters that
 QUEST_LV_0200_20150317_002413	You've defeated Elite Monsters as requested by Saliamonas. Let's go back to Saliamonas.
 QUEST_LV_0200_20150317_002414	Talk to the thief, Amanda
 QUEST_LV_0200_20150317_002415	Amanda is waiting for someone's help in the petrified city.
```
Comment (f-tsang) [inline QUEST_LV_0200.tsv]: (Same as above) Petrified City should be capitalized here.

**Comment by f-tsang** [comment]
Line 2400 is inconsistent with the surrounding lines. It's likely suppose to be lighting a brazier instead of burning incense. 'Fortress' shouldn't be capitalized. Referring to the Korean text, the burner/brazier seems to be at a bastion in the fortress.

**Comment by Flevalt** [comment]
capitalized Petrified City in the whole Quest_LV_0200.tsv.

**Comment by marikim** [comment]
Thank you :+1:

**Pull request merged**